modules/openshift-developer-cli-setting-and-unsetting-environment-variables.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/managing-environment-variables-in-odo.adoc
+
+[id="setting-and-unsetting-environment-variables._{context}"]
+
+= Setting and unsetting environment variables
+
+.Procedure
+
+* To set an environment variable in a component:
+----
```

Comment:
Please add + before the code snippet to align them with the instructions.

Reply:
Added in https://github.com/openshift/openshift-docs/pull/18478/commits/05d79f62fcd8edc1d07dab175bcdbb8d7a252382

Reply:
Added in https://github.com/openshift/openshift-docs/pull/18478/commits/05d79f62fcd8edc1d07dab175bcdbb8d7a252382